.github/workflows/publish-docs-to-s3.yml

```diff
@@ -31,7 +31,7 @@ on:  # yamllint disable-line rule:truthy
           eg: amazon common.messaging apache.kafka
 
         required: false
-        default: "all"
+        default: "all-providers"
```

Comment:
Whats the difference?
Generally speaking the all option would be rarely used. Most often at least one provider has rc2.

Reply:
"all" means "ALL" - provders, airflow, helm chart , docker-stack.

We never publish them all together.

Reply:
> Generally speaking the all option would be rarely used. Most often at least one provider has rc2.

It's not for RC2 - it's for publishing docs after release is done. Not sure what `rc2` has to do with it ?